consensus/scheme/rolldpos/rolldpos.go

```diff
@@ -235,6 +235,7 @@ type Builder struct {
 	encodedAddr      string
 	priKey           crypto.PrivateKey
 	chain            ChainManager
+	evmNetworkID     uint32
```

Comment:
Attach `blockDeserializer *block.Deserializer` instead of `evmID` in the struct as the implement  https://github.com/iotexproject/iotex-core/commit/b015a15ec6f9d47393a8329bb2caec70de69a555